addons/binding/org.openhab.binding.amazondashbutton/README.md

```diff
@@ -9,51 +9,47 @@ The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP and BO
 
 ## Install Native Library
 
-Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J.
+Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J. Pcap4J needs administrator/root privileges.
 
-### Ubuntu
+Source: [https://www.pcap4j.org/](https://www.pcap4j.org/)
+
+### Debian/Ubuntu
```

Comment:
As we were talking about in the other thread. I'd reduce the "big" headlines count. How about:

H3 OS Specific Notes
H4 Debian/Ubuntu
H4 CentOS
...